src/Entity/Timeline/Measure.php

```diff
@@ -267,14 +279,13 @@ public function isDeferred(): bool
         return self::STATUS_DEFERRED === $this->status;
     }
 
-    public function saveCurrentThemes(): void
-    {
-        $this->savedThemes = clone $this->themes;
-    }
-
     private function getSavedThemes(): Collection
     {
-        return $this->savedThemes ?? new ArrayCollection();
+        if (!$this->savedThemes) {
+            $this->savedThemes = new ArrayCollection();
```

Comment:
You don't need to do that if `new ArrayCollection` is in the constructor.

Reply:
Not when it comes from Doctrine.

Reply:
Doctrine give you a `PersistedCollection`.

Reply:
Yeah but this is not a collection managed by Doctrine